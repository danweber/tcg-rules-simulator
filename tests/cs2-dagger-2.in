VERSION:2
# [Your Turn] While you have a red Monster or Tamer, all your Monsters gain +1000 DP
# CS2-02 has 2K by default
FIELD2:Normal,CS2-02 Egg1,CS2-02,CS2-01 Egg2,CS2-02,CS2-01
HAND2:CS2-05
GAME:P2 T1 MAIN 8
DUMP:DP

ECHO: -- Play red mon, get +2000
CMD:Play CS2-05 Redrum
DUMP:DP

