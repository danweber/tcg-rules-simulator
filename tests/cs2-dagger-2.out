FIELD1:
FIELD2:Normal,CS2-02,2000DP Egg1,CS2-02,CS2-01,2000DP Egg2,CS2-02,CS2-01,2000DP
GAME:P2 T1 MAIN 8
ECHO: -- Play red mon, get +2000
FIELD1:
FIELD2:Normal,CS2-02,4000DP Egg1,CS2-02,CS2-01,4000DP Egg2,CS2-02,CS2-01,4000DP Redrum,CS2-05,7500DP
GAME:P2 T1 MAIN 3
